examples/bpk-component-skeleton/stories.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  BpkBodyTextSkeleton,
+  BpkCircleSkeleton,
+  BpkHeadlineSkeleton,
+  BpkImageSkeleton,
+} from '../../packages/bpk-component-skeleton/index';
+
+import {
+  BaseSkeletonExample,
+  ImageDefaultSizeWithDefaultStyleExample,
+  ImageDefaultSizeWithRoundedStyleExample,
+  ImageCustomNumberSizeWithDefaultStyleExample,
+  ImageCustomStringSizeWithRoundedStyleExample,
+  BodyTextSmallSizeExample,
+  BodyTextDefaultSizeExample,
+  BodyTextLargeSizeExample,
+  BodyTextCustomSizeExample,
+  CircleSmallSizeExample,
+  CircleDefaultSizeExample,
+  CircleCustomSizeExample,
+  HeadlineSmallSizeExample,
+  HeadlineDefaultSizeExample,
+  HeadlineLargeSizeExample,
+  HeadlineCustomSizeExample,
+  VisualTestExample,
+} from './examples';
+
+export default {
+  title: 'bpk-component-skeleton',
+  component: BpkImageSkeleton,
+  subcomponents: {
+    BpkBodyTextSkeleton,
+    BpkCircleSkeleton,
+    BpkHeadlineSkeleton,
+  },
+};
+
+export const Base = BaseSkeletonExample;
+
+export const ImageDefaultSizeWithDefaultStyle = ImageDefaultSizeWithDefaultStyleExample;
+export const ImageDefaultSizeWithRoundedStyle = ImageDefaultSizeWithRoundedStyleExample;
+export const ImageCustomNumberSizeWithDefaultStyle = ImageCustomNumberSizeWithDefaultStyleExample;
+export const ImageCustomStringSizeWithRoundedStyle = ImageCustomStringSizeWithRoundedStyleExample;
+
+export const BodyTextSmallSize = BodyTextSmallSizeExample;
+export const BodyTextDefaultSize = BodyTextDefaultSizeExample;
+export const BodyTextLargeSize = BodyTextLargeSizeExample;
+export const BodyTextCustomSize = BodyTextCustomSizeExample;
+
+export const CircleSmallSize = CircleSmallSizeExample;
+export const CircleDefaultSize = CircleDefaultSizeExample;
+export const CircleCustomSize = CircleCustomSizeExample;
+
+export const HeadlineSmallSize = HeadlineSmallSizeExample;
+export const HeadlineDefaultSize = HeadlineDefaultSizeExample;
+export const HeadlineLargeSize = HeadlineLargeSizeExample;
+export const HeadlineCustomSize =HeadlineCustomSizeExample;
+
+export const VisualTest = VisualTestExample;
+export const VisualTestWithZoom = VisualTest.bind({});
+VisualTestWithZoom.args = {
+  zoomEnabled: true
+};
```

Comment:
For this component I think a visual test is not going to be great as the snapshots will be inconsistent between runs and constantly cause Percy CI to fail for PRs.

So my suggestion here would be to remove these stories and we will just need to use manual checks to verify all is well

Reply:
That makes sense. The visual test has been removed.